### DISCUSSION TITLE
action with antivirus

### DISCUSSION THREAD

**Discussion by yreweb** [rspamd/rspamd#4431] [Q&A]
I try to find information regarding the _action_ parameter in antivirus.inc. Is it possible to use something else than _'reject'_ ?
Thanks.

**Comment by fatalbanana** [comment]
It should be possible to use any valid action including custom actions.

https://rspamd.com/doc/configuration/metrics.html

**Reply by yreweb** [reply]
Thanks. I didn't get it by myself in doc.
